src/diffusers/loaders/__init__.py

```diff
@@ -75,8 +75,10 @@ def text_encoder_attn_modules(text_encoder):
             "SanaLoraLoaderMixin",
         ]
         _import_structure["textual_inversion"] = ["TextualInversionLoaderMixin"]
+        _import_structure["ip_adapter"] = ["IPAdapterMixin"]
```

Comment:
I think this line isn't necessary? The key is overwritten in the next line.

Reply:
Removed, left over from merge conflict from SD3.5 IP adapter